PickNikRobotics/UR10e_welding_demo
# Servo as local planner demonstration

# Description

- Adds Servo as a local planner (following @sjahr's PR for that over in moveit2)
- Hardcodes the MTC global planner to do a relative Cartesian plan
- Adds a joint configuration for the start of the Cartesian path
- Allows Servo local planner to execute the trajectory at constant velocity, with user input to allow rotation of the last joint on-line

# How to test
Follow the updated README section `Hybrid Planning Demonstration`

Note this PR includes a dependency change in `moveit2`, so you should pull the correct version and build in order to test properly

# Next Steps
This is mostly a demonstration of the hybrid planning capabilities. I am not too sure we actually want to merge this in (or at least as is now with hardcoded stuff for the demo). @henningkayser do you have any next steps for this effort?
